app/Livewire/Chat.php

```diff
@@ -73,7 +72,7 @@ public function mount($id = null)
         // If ID is not null, we're in a thread. But if thread doesn't exist or doesn't belong to the user and doesn't match the session ID, redirect to homepage.
         if ($id) {
             $thread = Thread::find($id);
-            if (! $thread || (auth()->check() && $thread->user_id !== auth()->id()) || (! auth()->check() && $thread->session_id !== session()->getId())) {
+            if (!$thread || (auth()->check() && $thread->user_id !== auth()->id()) || (!auth()->check() && $thread->session_id !== session()->getId())) {
```

Comment:
It also seems that `./vendor/bin/pint` was not run, or it ran with wrong parameters, so we lost all spaces after the negations.

Reply:
It ran but maybe with the config.
This is still a WIP